modules/radio/react/lib/Radio.tsx

```diff
@@ -75,10 +75,18 @@ const RadioInputWrapper = styled('div')<Pick<RadioProps, 'disabled'>>({
   width: radioWidth,
 });
 
-/**
- * Note: `~ div:first-of-type` refers to `RadioBackground`
- * and was easier to use than a component selector in this case.
- */
+const RadioRipple = styled('span')<Pick<RadioProps, 'disabled'>>({
+  borderRadius: borderRadius.circle,
+  boxShadow: `0 0 0 0 ${colors.soap200}`,
+  content: '""',
+  height: radioHeight,
+  transition: 'box-shadow 150ms ease-out',
+  width: radioWidth,
+  position: 'absolute',
+  pointerEvents: 'none',
```

Comment:
Don't want the ripple to be clickable. It alternates from being behind the `RadioBackground` to in front depending on the state of the input.

Reply:
I think that's fine. The radio input is in the same position with a opacity of 0, right? If you think that's a questionable point, you could add a comment. Cypress will also catch errors here if we have a test that makes sure clicking on the input works. It would fail if something is preventing the click from reaching the input.

Reply:
Inline comment added. That's a good call-out.